andela/wg-a-team
# #152919330 Create authors field

#### What does this PR do?
It creates authors field and adds a foreign key
#### Description of Task to be completed?
The exercise model didn't have the authors field. This PR creates the authors' field and allows users to select the author of the exercise when adding the exercise.
#### How should this be manually tested?
After pulling this branch run 
```
invoke load_fixtures --settings-path=settings.py
./manage.py makemigrations
./manage.py migrate
./manage.py runserver
```
After logging in to the app under the exercises tab click exercises then add exercise. When you click the authors field the application should list all the authors in the application and in case this field is left empty the logged in user is added as the author of that exercise.
#### What are the relevant pivotal tracker stories?
[#152919330](https://www.pivotaltracker.com/story/show/152919330)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/15782137/34033415-308b18c6-e18b-11e7-92ab-5ff9701f324f.png)
